third_party/GSL/span.h

```diff
@@ -3539,7 +3539,7 @@ class span
 
 #else
 
-  template< size_t N
+  template< size_t \
```

Comment:
Hmmm.  This is an unexpected change.

Reply:
Hmm, I also didn't expect this change. Thanks so much for finding this.

Reply:
OK, if you are planning on making that tweak then maybe go ahead and make those local constexprs static.  We can tackle the rest of my comments in the next PR.